### ISSUE TITLE
[SCREENREADER]: Consider updating phone number documentation in design system, to include spacing in the aria-label

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#6997]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

**✔️ Consider** updating phone number documentation in design system, to include spacing in the aria-label. 

Currently some screen readers will read a phone number as "seven-hundred-and-three five-hundred-fifty-five minus one-thousand-two-hundred-and-twelve". The addition of the aria-label with the number spaced out like that makes screen readers spell out each number so the user can understand. Ideally, this technique would be included in the design system, so we can refer teams to it.

Trevor's note: we’ll need to petition Deque and/or W3C to remove that as a violation before axe bakes in WCAG 2.1.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to hear the phone, fax, and TTY numbers spelled out in a way I can understand.

## Environment

* Operating System: Windows
* Browser: all
* Screenreading device: JAWS / NVDA
* Server destination: all

## Possible Fixes (optional)

Update design system documentation for phone numbers to include an aria-label that spaces out the numbers, as in this example:
```html
<a href="tel:7035551212" aria-label="7 0 3. 5 5 5. 1 2 1 2.">(703) 555-1212</a>
```

## WCAG or Vendor Guidance, References

* [Accessibility and phone number formatting, That Dev Girl](https://thatdevgirl.com/blog/accessibility-phone-number-formatting)

* [Deque axe-core rule, label and name from content mismatch](https://dequeuniversity.com/rules/axe/3.3/label-content-name-mismatch)

**Comment by caw310** [comment]
@k80bowman should we retest to see if this has been addressed?

**Comment by laflannery** [comment]
@caw310 this is very old and tagged `sitewide` so popped up on our board. I'm closing this as it's no longer needed because aria-label is no longer recommended as part of the phone number markup and has in fact been removed from the guidance and the component.
